src/categorical_algebra/ACSetsGATsInterop.jl

```diff
@@ -22,6 +25,21 @@ function Schema(p::Presentation)
   BasicSchema(obs,homs,attrtypes,attrs)
 end
 
+function Schema(c::GATContext)
```

Comment:
If codecov can be believed, this function isn't being run at all. Is it intended to be here?

Reply:
It makes sense as a function to have, so I think I'll just add a test for it.